testing/kuttl/e2e/root-cert-ownership/01-check-owners.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Get a list of the current owners of the root ca cert secret and verify that
+  # both owners are listed.
+  - script: |
+      CURRENT_OWNERS=$(kubectl --namespace="${NAMESPACE}" get secret \
+      pgo-root-cacert -o jsonpath='{.metadata.ownerReferences[*].name}')
```

Comment:
I think it's easier to read if we indent this line that continues the line above.